release/tools/schema_gen.sh

```diff
@@ -101,7 +101,7 @@ case `whoami` in
     gangaat )
         export LOAD_PACKAGES='GangaTest:GangaAtlas' ;;
     gangalb )
-        export LOAD_PACKAGES='GangaTest:GangaLHCb' ;;
+        export LOAD_PACKAGES='GangaTest:GangaDirac:GangaLHCb' ;;
```

Comment:
Can we add GangaGaudi here too, just for completeness so we don't revisit and wonder why the change.
(the schema test should probably be one we port fairly soon so we can keep track in the integration tests I suppose)